src/module/iotjs_module_dns.c

```diff
@@ -94,8 +100,7 @@ static void AfterGetAddrInfo(uv_getaddrinfo_t* req, int status,
 
   iotjs_jargs_destroy(&args);
 
-  iotjs_getaddrinforeqwrap_destroy(req_wrap);
-  IOTJS_RELEASE(req_wrap);
+  iotjs_getaddrinforeqwrap_dispatched(req_wrap);
```

Comment:
I feel difficulty in reading very long word like `iotjs_getaddrinforeqwrap` without separator. What about using iotjs_getaddrinfo_reqwrap? There are many cases. As you said above comment, although it is not main change of this patch, if you would like to upload the patch of enhancing readability, I will be happy. Since your patches for C++ to C has been very big (in terms of line or in terms of file changed) and related all the modules tightly. I could not review all the details of your patch  in appropriate time. Sorry about that. Thank you.